perf/benchmark/runner/runner.py

```diff
@@ -93,6 +93,7 @@ def kubectl_exec(pod, remote_cmd, runfn=run_command, container=None):
 class Fortio:
     ports = {
         "http": {"direct_port": 8077, "port": 8080},
+        "tcp": {"direct_port": 8078, "port": 8078},
```

Comment:
Should direct_port be the same as port?

Also a bit odd "grpc" and "direct_envoy" are the same port... but not related to this pr?

Reply:
Not sure any side effect here, need @carolynhu help.